code/game/objects/items/devices/radio/radio.dm

```diff
@@ -541,6 +541,8 @@
 	subspace_switchable = 1
 	dog_fashion = null
 
+/obj/item/device/radio/borg/emp_act(severity)	//keeps bad things from happening
+

```

Comment:
I'm dubious of empty definitions ever since a misplaced line gave xeno larvae telekinesis.